mlflow/pyfunc/__init__.py

```diff
@@ -1583,7 +1583,7 @@ def batch_predict_fn(pdf, params=None):
                 model_path = os.path.join(
                     tempfile.gettempdir(),
                     "mlflow",
-                    hashlib.sha1(model_uri.encode()).hexdigest(),
+                    sha1(model_uri.encode()).hexdigest(),
```

Comment:
Not a blocker, but can we use `insecure_hash.sha1` to emphasize we're using our own module?

Reply:
Sure, that sounds cleaner